controllers/articles.controller.js

```diff
@@ -7,3 +7,15 @@ exports.getArticlesByIDCon = (req, res, next) => {
     })
     .catch((next))
 }
+
+exports.getArticlesCon = (req, res, next) => {
+    const {sort_by} = req.query
```

Comment:
At this point there wouldn't be a query on this endpoint. Now eventually there will be, the sort order should be default on just /api/articles